www/src/pages/components/forms.js

```diff
@@ -410,11 +410,21 @@ export default withLayout(function FormControlsSection({ data }) {
       <p>
         Bootstrap scopes the <code>:valid</code> and <code>:invalid</code>{' '}
         styles to parent <code>.was-validated</code> class, usually applied to
-        the <code>Form</code> (you can use the <code>validated</code> prop as a
-        shortcut). Otherwise, any required field without a value shows up as
-        invalid on page load. This way, you may choose when to activate them
+        the <code>{'<Form>'}</code> (you can use the <code>validated</code> prop
+        as a shortcut). Otherwise, any required field without a value shows up
+        as invalid on page load. This way, you may choose when to activate them
         (typically after form submission is attempted).
       </p>
+      <p>
+        For custom Bootstrap form validation messages, you’ll need to add the{' '}
+        <code>noValidate</code> boolean attribute to your{' '}
+        <code>{'<Form>'}</code>. This disables the browser default feedback
+        tooltips, but still provides access to the form validation APIs in
+        JavaScript. Try to submit the form below; our JavaScript will intercept
+        the submit button and relay feedback to you. When attempting to submit,
+        you’ll see the <code>:invalid</code> and <code>:valid</code> styles
+        applied to your form controls.
+      </p>
```

Comment:
This just looks like a direct copy/paste of the [bootstrap docs](https://getbootstrap.com/docs/4.0/components/forms/#custom-styles)?

I don't think it's a good idea to add this, if anything we can just link to this section of their docs.

Reply:
Yeah it's mostly a copy/paste with some minor edits for React.  This came up in #5262, so I thought it would help to clear up any confusion for users.  I'm fine with linking to the official docs if that's the way to go.

Reply:
I think it's ok to restate docs here if relevant. TBH most of our docs are copies of upstream any :P

Reply:
Idk I still think it’s a hassle to have duplicate info in here, since the upstream docs might change and we’d have to update this once it does 🤷

Maybe just a brief summary followed by an external link to the upstream docs?

Reply:
I did notice that a lot of info were copies from the official docs with respective React info substituted in (ie. previous paragraph), so I thought I'd follow suit with that.  

I can probably chop the text in half to make it shorter and put in the official link?

Reply:
I generally agree with not duplicating info, but this isn't likely to change because it's not bootstrap specific, it's surfacing and noting normal HTML behavior that needs to be considered here. Also people don't really click through to go read the bootstrap docs, or often understand why they would be relevant to RB.  I would just slightly reword the upstream notes a bit.

Reply:
I would rework this to something like "Watch out! Browsers, provide their own validation UI by default on `form`s. You can disable the default UI by adding the HTML `noValidate` attribute to your Form/form element."

Reply:
Thanks for the feedback @bpas247 & @jquense. I've checked in new wording.